airflow-ctl/src/airflowctl/api/operations.py

```diff
@@ -243,8 +243,11 @@ def create_event(
     ) -> AssetEventResponse | ServerResponseError:
         """Create an asset event."""
         try:
+            # Ensure extra is an object; DB/UI choke on null
+            if asset_event_body.extra is None:
+                asset_event_body.extra = {}
             self.response = self.client.post(
-                "assets/events", json=_date_safe_dict_from_pydantic(asset_event_body)
+                "/assets/events", json=_date_safe_dict_from_pydantic(asset_event_body)
```

Comment:
I mentioned a couple of times that this should not actually align with multiple of them, and we should use the other way around. That part is working. We don't need to update it
```suggestion
                "assets/events", json=_date_safe_dict_from_pydantic(asset_event_body)
```